extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -256,13 +255,17 @@ class QueueProcessor extends EventEmitter {
             });
             this._consumer.on('metrics', data => {
                 // i.e. data = { my-site: { ops: 1, bytes: 124 } }
-                const filteredData = Object.keys(data).filter(key =>
-                    key === this.site).reduce((store, k) => {
-                        // eslint-disable-next-line no-param-reassign
-                        store[k] = data[this.site];
-                        return store;
-                    }, {});
-                this._mProducer.publishMetrics(filteredData,
+
+                // NOTE: No need to filter data since we are accepting
```

Comment:
@jonathan-gramain I removed this filter. Let me know if you have any concerns :)

Reply:
This change lgtm. The filter was originally added so we don't duplicate metrics for every QueueProcessor that receives the metrics event. Since we only use a single QueueProcessor for all sites now, I think this change looks good. Thanks!

Reply:
Then, can you remove the commented code? Thanks!